### PR TITLE
ZEPPELIN-1597. Change cdh spark/hadoop dependency to apache spark/hadoop

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
I see several times of travis failure due to downloading cdh spark/hadoop for livy interpreter. This may be due to cdh repository issue, so in this PR I change it to apache spark/hadoop.

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-1597

### How should this be tested?
No test added

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
@prabhjyotsingh @felixcheung Please help review.

File: livy/pom.xml [hunk 1]
```diff
@@ -41,6 +41,8 @@
         <assertj.version>1.7.0</assertj.version>
         <mockito.version>1.9.5</mockito.version>
         <livy.version>0.2.0</livy.version>
+        <spark.version>1.5.2</spark.version>
```
Comment (zjffdu) [inline livy/pom.xml]: livy 0.2 use spark 1.5 by default. https://github.com/cloudera/livy/blob/branch-0.2/pom.xml

**Comment by minahlee** [comment]
LGTM. I will merge it as hotfix if there is no more comment.

**Comment by anthonycorbacho** [comment]
@minahlee yes please merge it asap this a total war zone here

**Pull request closed without merging**